bin/helm-include.sh

```diff
@@ -95,8 +95,39 @@ function helmRepoAdd {
   fi
 }
 
+function get_helmchart_reference {
+  local chart_repository chart_name chart_version
+
+  chart_repository=$1
+  chart_name=$2
+  chart_version=$3
+
+  if [ -z "$chart_repository" ]; then
+     echo "'ERROR: Helm chart repository not specified'"
+     return 1
+  elif [ -z "$chart_name" ]; then
+     echo "'ERROR: Helm chart name not specified'"
+     return 1
+  elif [ -z "$chart_version" ]; then
+     echo "'ERROR: Helm chart version not specified'"
+     return 1
+  else
+     #all parms have values
+     :
+  fi
+
+  if [ "$AIRGAP_HELM_FORMAT"  == "tgz" ]; then
+      echo "${AIRGAP_HELM_REPO}/${chart_name}-${chart_version}.tgz"
```

Comment:
We should probably test to make sure that a file that matches the pattern exists in that spot before we get too far in the process.

Reply:
I added a check in airgap-include.sh that confirms the AIRGAP_HELM_REPO is an existing directory when AIRGAP_HELM_FORMAT is "tgz".   That will trigger a failure very early in the deployment process if that environment variable isn't set properly.   I did NOT add a file exists check here because Helm actually already emits a good error message when that happens; adding our own would have been redundant.